### DISCUSSION TITLE
Are there going to be alternatives to Zed AI after launch?

### DISCUSSION THREAD

**Discussion by mak448a** [zed-industries/zed#17265] [Feature Requests]
Title. Are there going to be third party extensions that replace the functionality?

**Comment by widersky** [comment]
Zed already supports a million alternative language models, what more do you expect?

**Reply by mak448a** [reply]
It won't be free anymore after launch. See https://zed.dev/blog/zed-ai

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
